contributing.md

```diff
@@ -2,17 +2,18 @@
 # Contributing
 
 ### Terms
-  * Upstream: the repository from the ```original_author```
-  * Origin: the forked repository under your ```account```
+<<<<<<< HEAD
+  * Upstream: the repository from the `original_author`
+  * Origin: the forked repository under your `account`
+<br>
```

Comment:
And this `<br>` too.

Reply:
@sara-02 I am not sure what is happening, but I did remove those two lines locally, this is really weird